misc/python/materialize/mzcompose/__init__.py

```diff
@@ -513,6 +513,12 @@ def sql(self, sql: str) -> None:
                 print(f"> {statement}")
                 cursor.execute(statement)
 
+    def query(self, sql: str) -> tuple[list[Any]]:
```

Comment:
Ideally this would be `sql_query` and the other one would be `sql_execute`. Renaming `c.sql` would be a hassle, but WDYT about at least calling this one `sql_query`?

Reply:
Done.